components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -108,8 +106,8 @@ private BuildValidationData fetchBuildScanData(int index, URL buildScanUrl, Cust
                 "",
                 null,
                 Collections.emptyMap(),
-                Duration.ZERO,
-                BigDecimal.ZERO);
+                null,
+                null);
         }
```

Comment:
I think we could improve the readability here by creating a new constructor on BuildValidationData that only accepts the buildScanId and baseUrl, and otherwise sets all of the other fields equal to "no value" defaults.

Reply:
We'll do this after the move of the module.